model/distro/distro.go

```diff
@@ -281,9 +281,15 @@ func ValidateBootstrapAndCommunicationMethods(bootstrap string, communication st
 	catcher := grip.NewBasicCatcher()
 	catcher.Add(ValidateBootstrapMethod(bootstrap))
 	catcher.Add(ValidateCommunicationMethod(communication))
-	if bootstrap == BootstrapMethodLegacySSH && communication != CommunicationMethodLegacySSH ||
-		communication == CommunicationMethodLegacySSH && bootstrap != BootstrapMethodLegacySSH {
-		catcher.Add(fmt.Errorf("'%s' and '%s' is not a valid bootstrap and communication combination - legacy SSH is incompatible with non-legacy functionality", bootstrap, communication))
+	switch bootstrap {
+	case BootstrapMethodLegacySSH:
+		if communication != CommunicationMethodLegacySSH {
+			catcher.Add(fmt.Errorf("bootstrapping hosts using legacy SSH is incompatible with non-legacy host communication"))
```

Comment:
you only need to use Errorf if you're doing something sprintf-ish, use errors.New() otherwise. 

(also in general prefer errors.Errorf from pkg/errors over fmt.Errorf or) 

having said that, the catcher interface now, has a .New() method that takes a string.